src/components/ItemTable/ItemAvailability.tsx

```diff
@@ -100,15 +101,7 @@ const ItemAvailability = ({ item }: ItemAvailabilityProps) => {
         <Button
           id="contact-librarian"
           buttonType="link"
-          sx={{
-            display: "inline",
-            fontWeight: "inherit",
-            fontSize: "inherit",
-            p: 0,
-            height: "auto",
-            textAlign: "left",
-            minHeight: "auto",
-          }}
+          sx={BUTTON_LINK_STYLES}
```

Comment:
Edwin added that comment in the advanced search page but I believe it should always be rendered to control the focus for accessibility. Just amended the comment.

Reply:
I ended up scrapping this, since i was just trying to cut down duplication since we override the button styles to match the DS link styles in one other place. This should be addressed on the DS level, will add TODO comments and bring it up in the next DS meeting.